internal/core/data/main.go

```diff
@@ -48,7 +50,7 @@ func Main(ctx context.Context, cancel context.CancelFunc, router *mux.Router, re
 	//      ....
 	//      flags.Parse(os.Args[1:])
 	//
-	flags := commandline.NewDefaultCommonFlags("")
+	flags := flags.New()
```

Comment:
In other services you changed `flags` to `f`. Should be consistent.

Reply:
Can't believe I missed that.  Thank you for catching it.